luaui/Widgets/gui_sensor_ranges_los.lua

```diff
@@ -11,11 +11,12 @@ function widget:GetInfo()
 end
 
 -------   Configurables: -------------------
-local rangeColor = { 0.9, 0.9, 0.9, 0.24 } -- default range color
+local rangeColor = { 0.9, 0.9, 0.9, 0.74 } -- default range color
```

Comment:
Why was this changed, is the appearence different?

Reply:
Oh yes that change wasn't necessary and can be reverted!